interceptor/tracing_interceptor_test.go

```diff
@@ -0,0 +1,63 @@
+package interceptor_test
+
+import (
+	"context"
+	"testing"
+
+	"go.temporal.io/sdk/interceptor"
+	"go.temporal.io/sdk/internal/interceptortest"
+)
+
+type testTraceInterceptor struct {
+	interceptor.BaseTracer
+	t *testing.T
+}
+
+type testSpan struct{}
+
+func (t testSpan) Finish(options *interceptor.TracerFinishSpanOptions) {}
+
+type testSpanRef struct{}
+
+func (t testTraceInterceptor) Options() interceptor.TracerOptions {
+	return interceptor.TracerOptions{
+		SpanContextKey:       "test-tracer",
+		HeaderKey:            "test-tracer",
+		DisableSignalTracing: false,
+		DisableQueryTracing:  false,
+	}
+}
+
+func (t *testTraceInterceptor) UnmarshalSpan(m map[string]string) (interceptor.TracerSpanRef, error) {
+	return testSpanRef{}, nil
+}
+
+func (t *testTraceInterceptor) MarshalSpan(span interceptor.TracerSpan) (map[string]string, error) {
+	return map[string]string{}, nil
+}
+
+func (t *testTraceInterceptor) SpanFromContext(ctx context.Context) interceptor.TracerSpan {
+	return testSpan{}
+}
+
+func (t *testTraceInterceptor) ContextWithSpan(ctx context.Context, span interceptor.TracerSpan) context.Context {
+	return ctx
+}
+
+func (t *testTraceInterceptor) StartSpan(options *interceptor.TracerStartSpanOptions) (interceptor.TracerSpan, error) {
+	// Require start time to be set
+	if options.Time.IsZero() {
+		switch options.Operation {
+		case "RunWorkflow", "RunActivity":
+		// Do nothing; the test env doesn't set these at the moment.
```

Comment:
I haven't had time to look into this more, but it seems that workflow and activity start times aren't being set in workflow and activity info structs passed to interceptors in the test environment.